Functions.cmake

```diff
@@ -87,3 +87,17 @@ function(install_webplugin _dir)
   file(GLOB _userguides "${_dir}/userguide/*.md")
   set_property(GLOBAL APPEND PROPERTY USERGUIDES "${_userguides}")
 endfunction(install_webplugin)
+
+# Finds the absolute paths for the given Boost libraries
+# Use variable arguments for the Boost libraries to link
+function(find_boost_library)
```

Comment:
To keep ourselves to CMake's DSL's naming convention, the function should have a plural name: `find_boost_libraries`.

Reply:
Updated accordingly.